timetracker/timetracker/urls.py

```diff
@@ -18,11 +18,35 @@
 from django.http import HttpResponse
 
 
-def hello_world(request):
-    return HttpResponse('Hello World')
+def hello_world(request, name):
+   name = request.GET.get('name')
+   #
+   # Start of corner cutting
+   #
+   ghetto_template = "<html>\n\
```

Comment:
@gazwald you can use a triple quote `"""` to create multi line strings in Python.